src/mrack/outputs/ansible_inventory.py

```diff
@@ -206,4 +209,6 @@ def create_output(self):
         """Create the target output file."""
         inventory = self.create_inventory()
         save_yaml(self._path, inventory)
+        if self._path:
+            logger.info(f"Created: {self._path}")
```

Comment:
In these, could you please use lazy formatting?
`W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)`
I future pylint will not complain.

Reply:
Is it actually needed? By rough estimates, 2/3 of logger.info calls in the code base are using this method. I believe there is none with lazy formatting. Atm. it will make it more inconsistent with rest of code base.

Also, I believe that with W1203 we would be imposing unecessary purity over readability.  W1203 is about performance and performance impact with f-strings is almost 0.

Reply:
Let us mark it as resolved then

Reply:
Personally, I have no opinion about this, but recently I was faced with the same dilemma on another project and found this: https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles

One _possible_ reason for using `%`: mrack can be imported as a module and the main project can independently decide how logs are handled (Assuming backwards compatibility).